### PR TITLE
Set colors for all circleci statuses

### PR DESCRIPTION
**Pull request by tusbar**

### CODE REVIEW

**Comment by amio** [comment]
Awesome! I was thinking doing this 🎉

File: libs/live-fns/circleci.js [hunk 1]
```diff
@@ -10,6 +10,36 @@ module.exports = async function (vcsType, username, project, branch) {
...
+    case 'not_running':
+      return 'grey'
+
+    case 'queued':
+    case 'scheduled':
+      return 'yellow'
+
+    case 'retried':
+    case 'running':
+      return 'blue'
```
Comment (olstenlarck) [inline libs/live-fns/circleci.js]: I think this should be orange or somthing like that.

Comment (tusbar) [reply libs/live-fns/circleci.js]: I was thinking the same thing, but orange looked too much like "failed". 

Cyan wouldn’t be too bad.

Comment (amio) [reply libs/live-fns/circleci.js]: Yeah, right. 😅

Prefer cyan, hint some feels about "towards stable" like versions.

Comment (olstenlarck) [reply libs/live-fns/circleci.js]: omg

Comment (amio) [reply libs/live-fns/circleci.js]: Any color proposal? Or let it be cyan.

**Pull request merged**